python/lsst/daf/butler/cli/cliLog.py

```diff
@@ -273,3 +284,24 @@ def _recordComponentSetting(cls, component):
         component levels."""
         componentSettings = cls.ComponentSettings(component)
         cls._componentSettings.append(componentSettings)
+
+    @classmethod
+    def replayConfigState(cls, configState):
+        """Re-create configuration using configuration state recorded earlier.
+
+        Parameters
+        ----------
+        configState : `list` of `tuple`
+            Tuples contain a method as first item and arguments for the method,
+            in the same format as ``cls.configState``.
+        """
+        if cls._initialized or cls.configState:
+            # Already initialized, do not touch anything.
+            log = logging.getLogger(__name__.partition(".")[2])
```

Comment:
In all other butler code we use `__name__` and don't strip the `lsst.` (I see it's actually two places because something else crept in as well)

Reply:
I'm trying to follow recommendations that say `lsst` should be dropped: https://developer.lsst.io/stack/logging.html#logger-names. I can certainly change that to just `__name__` if this is preferred.

Reply:
butler is special and has been for a long time -- if you drop it people will have to use lsst.daf.butler and daf.butler to turn on debug logging .

Reply:
OK, I'll replace that and other cases if I find any.

Reply:
The only other one was in core/utils.py I think

Reply:
Replaced 3 instances in a new commit.